sleap_io/io/nwb.py

```diff
@@ -0,0 +1,282 @@
+"""Functions to write and read from the neurodata without borders (NWB) format. 
+"""
+from tkinter import Label
```

Comment:
What's this for?

Reply:
This got imported because vscode was trying to be too smart. Removed.